index.html

```diff
@@ -2374,21 +2394,6 @@ <h3>
 consumption rules</a> in the [[INFRA]] specification.
         </p>
 
```

Comment:
```suggestion
        <p>
Note that the value of the <code>@context</code> object member will be ignored, if present. That is, this field will not have additional processing applied to its value and will be added verbatim to the abstract data model.
        </p>
```

Reply:
I'm also ok with this version of the note. We might want to mention that it follows the same rules as any other representation syntax (it's not special), but that can be done editorially.

Reply:
@selfissued this suggestion is an attempt to address your concerns, can you provide feedback so we can move this forward?

Reply:
I can't seem to make a suggestion atop the suggestion, so I'll just write it here.
```suggestion
Note that no additional processing will be applied to the value of the 
<code>@context</code> object member, if present. The original value will 
be added verbatim to the abstract data model.
```